incubator/java-spring-boot2/README.md

```diff
@@ -28,14 +24,21 @@ The default template provides a `pom.xml` file that references the parent POM de
 
     This will initialize a Spring Boot 2 project using the microservice template.
 
-2. After your project has been initialized you can then run your application using the following command:
+1. Once your project has been initialized you can then run your application using the following command:
 
     ```bash
     appsody run
     ```
 
     This launches a Docker container that will run your application in the foreground, exposing it on port 8080. The application will be restarted automatically when changes are detected.
 
+1. You should be able to access the following endpoints that are exposed by your template  application by default:
```

Comment:
V minor - is the following slightly clearer?

You should be able to access the following endpoints, as they are exposed by your template application by default: